cli/command/stack/properties.go

```diff
@@ -0,0 +1,86 @@
+package stack
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/docker/cli/opts"
+	"github.com/docker/stacks/pkg/types"
+)
+
+func substituteProperties(input *types.ComposeInput, workingDir string) error {
```

Comment:
Why is this here ? Why not reusing code from `cli/cli/compose` package ? This adds yet another implementation of this variable substitution…